npm-shrinkwrap.json

```diff
@@ -796,7 +796,7 @@
         },
         "util": {
           "version": "0.10.3",
-          "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
+          "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
```

Comment:
Do we need to change the build script to generate npm-shrinkwrap.json? What will impact the future new dependency introduced?